packages/webpack/package.json

```diff
@@ -52,7 +52,6 @@
     "sass-loader": "^12.2.0",
     "source-map-loader": "^3.0.0",
     "style-loader": "^3.3.0",
-    "stylus": "^0.59.0",
     "stylus-loader": "^7.1.0",
```

Comment:
Shouldn't this be removed as well?

Reply:
Removed

Reply:
Ah, I removed it from the root `package.json`. This one is here for now until Nx v18. This PR removes the ability to generate new project with Stylus. The `stylus-loader` is still used for now for projects that have not migrated.

Reply:
removed